changelog.md

```diff
@@ -5,3 +5,6 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+### Changed
+- Make DB Client class isolated
```

Comment:
```suggestion
- Make JDBC Client class isolated.
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-java.jdbc/pull/175/commits/c7bfa6b487a3c7a3f3dba6af9a91eaaa114bb2fa